services/wallet/api.go

```diff
@@ -44,6 +44,10 @@ func (api *API) GetWalletToken(ctx context.Context, addresses []common.Address)
 	return api.reader.GetWalletToken(ctx, addresses)
 }
 
+func (api *API) GetCachedWalletTokens(ctx context.Context) (map[common.Address][]Token, error) {
```

Comment:
That can be quite confusing that the market data won't be send, I would try to find a specific name for this. Or add it

Reply:
I would add it in a separate PR, for now probably best to change the name then (`GetCachedWalletTokensWithoutMarketData` ? a bit verbose but very explicit :) )

Reply:
Find by me, I don't have anything better